web/src/components/SearchBox.vue

```diff
@@ -215,9 +221,6 @@ watch(
 );
 
 onMounted(() => {
-  store.updateConverters().catch((err: string) => {
```

Comment:
This was done for the autocomplete suggestions when typing queries, not sure why it was moved to the Results component. I'd say it's better kept in the searchbox since that component is on the screen all the time and the websocket handles updates over time, so updating the converters everytime you look at search results seems unnecessary.